pkgs/tools/security/sha1collisiondetection/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchFromGitHub, libtool }:
+
+stdenv.mkDerivation  rec {
+  pname = "sha1collisiondetection";
+  version = "git-20170221";
```

Comment:
Well, the second part of the announcement is that there is now code from the science community that everyone can use to counter all known negative issues with the known weakness - and avoid all suspicion. So this is an actual fix and improvement - by a full and independent reimplementation. For all intents and purposes, this is the 'new' sha1sum and the old one should be removed. It just has a different name, because someone else wrote it. The old sha1sum is broken, the new one is not (and in fact is pretty damage proof). When using the new sha1collisiondetection tool, you are safe against this whole class of attacks. That is new.